src/analysisd/labels.c

```diff
@@ -39,12 +40,11 @@ int labels_init() {
 
 /* Find the label array for an agent. Returns NULL if no such agent file found. */
```

Comment:
Typo: "...agent **file** ..."

Reply:
I decided to change the whole comment and make use of Doxygen syntax. Please, check it on the `labels.h` header.